plugin/phpactor.vim

```diff
@@ -469,6 +500,51 @@ function! phpactor#_rpc_dispatch(actionName, parameters)
         return
     endif
 
+    " >> update file source
+    "
+    " NOTE: This method currently works on a line-by-line basis as currently
+    "       supported by Phpactor.
```

Comment:
Expand on this explanation - the Diff implementation we use (from PHPUnit) only works on a line-by-line basis, so we are always removing and adding lines, this means that same-line edits might leave the cursor at the wrong position.